### PR TITLE
Fourth, fifth and sixth task

### PR DESCRIPTION
**Pull request by mgandore**

### CODE REVIEW

File: tasks/src/app/interactive-list/interactive-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Component } from '@angular/core';
+
+@Component({
+	selector: "interactive-list",
+	templateUrl: "./interactive-list.component.html",
+	styleUrls: ["./interactive-list.component.css"]
+})
+
+export class InteractiveList {
```
Comment (danielursu) [inline tasks/src/app/interactive-list/interactive-list.component.ts]:
```suggestion
export class InteractiveListComponent {

```

Comment (danielursu) [reply tasks/src/app/interactive-list/interactive-list.component.ts]: use F2 in order to replace in all project

File: tasks/src/app/interactive-list/interactive-list.component.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<button (click)="toggleList()">{{isListShowing ? "Hide" : "Show"}}</button>
+<ol *ngIf="isListShowing">
+	<li *ngFor="let name of names; let i = index" [ngClass]="i < 2 ? 'styled' : 'unstyled'">{{name}}</li>
```
Comment (danielursu) [inline tasks/src/app/interactive-list/interactive-list.component.html]:
```suggestion
	<li *ngFor="let name of names" [ngClass]="name === 'Mihai' ? 'styled' : null">{{name}}</li>

```

File: tasks/src/app/interactive-list/interactive-list.component.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.styled {
```
Comment (danielursu) [inline tasks/src/app/interactive-list/interactive-list.component.css]:
```suggestion
.highlight-name {

```

**Review by danielursu — CHANGES_REQUESTED** [review]

File: tasks/src/app/interactive-list/interactive-list.component.html (hunk 2 above)
Comment (danielursu) [reply tasks/src/app/interactive-list/interactive-list.component.html]: or you can have
'highlight-name': name === 'Catalin'

File: tasks/src/app/interactive-list/interactive-list.component.html [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<button (click)="toggleList()">{{isListShowing ? "Hide" : "Show"}}</button>
+<ol *ngIf="isListShowing">
+	<li *ngFor="let name of names; let i = index" [ngClass]="{'highlight-name' : name === 'Mihai'}">{{name}}</li>
```
Comment (danielursuu) [inline tasks/src/app/interactive-list/interactive-list.component.html]:
```suggestion
	<li *ngFor="let name of names" [ngClass]="{'highlight-name' : name === 'Mihai'}">{{name}}</li>
```

**Review by danielursuu — APPROVED** [review]

**Review by danielursuu — APPROVED** [review]

File: tasks/src/app/interactive-list/interactive-list.component.html [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<button (click)="toggleList()">{{isListShowing ? "Hide" : "Show"}}</button>
+<ol *ngIf="isListShowing">
+	<li *ngFor="let name of names; let i = index" [ngClass]="{'highlight-name' : name === 'Mihai'}"
+		[ngStyle]="{'font-size' : i % 2 == 0 ? '30px' : null}">{{name}}</li>
```
Comment (danielursu) [inline tasks/src/app/interactive-list/interactive-list.component.html]:
```suggestion
		[ngStyle]="{'font-size' : i % 2 === 0 ? '30px' : null}">{{name}}</li>

```

**Review by danielursu — APPROVED** [review]

**Pull request merged**